frontend/app/forms/inputs/group-select.component.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import { useCss } from "kremling";
+import { get } from "lodash";
+import { DataContainerContext } from "../data-container.component";
+
+export default function GroupSelect(props) {
+  const dataContext = React.useContext(DataContainerContext);
+  const value = get(dataContext.data, props.dataKey);
+  const scope = useCss(css);
+
+  return (
+    <div className="web-form-input" {...scope}>
+      <div className="select-label">{props.label}</div>
+      <select
+        value={value}
+        onChange={evt => dataContext.setData(props.dataKey, evt.target.value)}
+      >
+        <option value="nothingSelected">Please select an option.</option>
```

Comment:
Do the PDFs still render when `nothingSelected` is the option that was chosen by the user?